src/shared/bridge.js

```diff
@@ -16,6 +16,8 @@ export default class Bridge {
     this.channelListeners = {};
     /** @type {Array<(channel: RPC) => void>} */
     this.onConnectListeners = [];
+
+    this.on('connect', cb => cb());
```

Comment:
This could do with a comment here indicating why 'connect' is being registered with an apparently no-op listener.

Reply:
I added a comment about the callback.